core/src/main/java/com/radixdlt/messaging/core/MessageCentralImpl.java

```diff
@@ -222,13 +224,36 @@ public void close() {
   @Override
   public void send(NodeId receiver, Message message) {
     final var event = new OutboundMessageEvent(receiver, message, System.nanoTime() - timeBase);
-    if (!outboundQueue.offer(event) && outboundLogRateLimiter.tryAcquire()) {
-      log.error("Outbound message to {} dropped", receiver);
+    if (outboundQueue.offer(event)) {
+      final String typeForMetrics;
```

Comment:
(readability only)

in metrics:
`public enum OutboundMessageType {`

here:
`OutboundMessageType::from(message)`

Reply:
hmh there are quite a few of these messages, not sure if it's worth this huge match statement?

Reply:
_(happy to add it in a follow up PR if you think it's worth it_)